.circleci/config.yml

```diff
@@ -40,12 +40,12 @@ commands:
               --volume=$HOME/.cache/go-build/:/root/.cache/go-build \
               --volume=$HOME/go/pkg/mod/:/go/pkg/mod \
               --volume=$HOME/project/.ssh:/root/.ssh \
-              weaveworks/eksctl-build:$(cat .docker/image_tag) << parameters.cmd >>
+              sayboras/eksctl-build:$(cat .docker/image_tag) << parameters.cmd >>
           no_output_timeout: << parameters.timeout >>
   docker-pull-build-image:
     description: Docker pull image
     steps:
-      - run: docker pull "weaveworks/eksctl-build:$(cat .docker/image_tag)"
+      - run: docker pull "sayboras/eksctl-build:$(cat .docker/image_tag)"
```

Comment:
Is using `sayboras` as the Docker org. expected? (The commit introducing the change says "_Update image name for testing purpose_" so I am guessing this is an omission & needs to be removed before we merge?)

Reply:
@marccarre you are right. I already updated docker hash, but I don't have the permission to push to `weaveworks` docker hub, so just use my docker account for testing purpose in CI.

Might need one of maintainers to actually check out the branch and update Docker image. Thanks.

Reply:
Let me revert this change, however the build will fail later :)